src/exercise/02/02.py

```diff
@@ -0,0 +1,30 @@
+import requests
+
+
+def start():
+    print(
+        """
+    Welcome to Pokémon Finder!
+    --------------------------
+  """
+    )
+
+
+def get_choice():
+    return input("Which Pokémon do you want to know about...\n")
+
+
+def fetch_pokemon(choice):
+    req = requests.get("https://pokeapi.co/api/v2/pokemon/" + choice.lower())
```

Comment:
Yep, what @frenetic said, fstrings were a awesome addition in python 3

Reply:
so you could have:
```python
url = "https://pokeapi.co/api/v2/pokemon/"
requests.get(f"{url}{choice.lower()}")
```

or 
```python
url = "https://pokeapi.co/api/v2/pokemon/"
choice = choice.lower()
req = requests.get(f"{url}{choice}")

Reply:
Oh, good! Interpolation! I was like - "how does this not exist in Python?!"

Reply:
Done.